### ISSUE TITLE
Update web site submission guidelines

### ISSUE THREAD

**Issue by cmcqueen** [tLDP/LDP#8]
I'd like to submit an improvement to the Serial HOWTO. But what's the current process?

* The web site [mailinfo page](http://www.tldp.org/mailinfo.html) says "Remember that we prefer GitHub for new submits, followed by a message to submit@en.tldp.org!" But it provides no link to GitHub or any further description of the process
* The web site [author guide](http://www.tldp.org/LDP/LDP-Author-Guide/html/process.html) doesn't mention this GitHub project.
* I see there have been pull requests on this GitHub project. But they don't appear in the [recent changes](http://www.tldp.org/sorted_howtos.html) page, and looking at the recent [Bash Prog](https://github.com/tLDP/LDP/commit/603bdbd8b680dbaecd0af3f2a2adfa1427e02968) change, it seems it has not resulted in an update to the [corresponding page on the web site](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-3.html#ss3.4).

You might attract more contributions if the submission process was more transparent.

**Comment by ser** [comment]
You are absolutely correct, all that documents require update. We need more volunteers for it.

**Comment by mkomarinski** [comment]
We're working on these issues.  The LAG is being updated to reference git and GitHub and we're reviewing the documentation generation process so things get updated a bit faster.  If you'd like to volunteer to assist, you can join the discuss@en.tldp.org mailing list and we can certainly throw some tasks your way.

**Comment by cmcqueen** [comment]
I am not able to commit to volunteering on an ongoing basis. At this stage I just want to make a specific contribution to the Serial HOWTO. The GitHub pull request mechanism is good for these sort of one-off contributions with low administrative overheads.

**Comment by martin-a-brown** [comment]
Greetings Craig,

>I am not able to commit to volunteering on an ongoing basis. At 
>this stage I just want to make a specific contribution to the 
>Serial HOWTO. The GitHub pull request mechanism is good for these 
>sort of one-off contributions with low administrative overheads.

Yes, completely agreed.  Feel free to clone and update--just keep in 
mind that we hope to involve current author(s) of the documents 
before accepting pull requests.  The most recent listed author of 
the Serial-HOWTO (written in Linuxdoc format, it seems) is David 
Lawyer <dave@lafn.org>.

We would prefer that the most-recent listed author 
generate/authorize the pull request; credited correctly to you, of 
course!

Could you contact him?  If that avenue fails, please let us know.

And, lastly, thank you for your interest in adding to and/or 
correcting this documentation.

-Martin

-- 
Martin A. Brown
http://linux-ip.net/

**Comment by sergiocollado** [comment]
Hi, same on here, I want to contribute by traslating, but Im unable to find the last autor, ... ¿any sugenstion?

Thansk a lot,

Regards,

Sergio

**Comment by jordialcoister** [comment]
Sorry, but Sergio's comment poped up a question on my head... How are the translations managed? I mean, the spanish translations for example were (still are, it seems) managed on a completely separate project and the docs can't be retrieved (I think) from the TDLP repositories, but for the project's... In the other hand there are various $LANGUAGE-howto's on your repository... 

I'd sure like to translate some docs to my language (valencian/catalan) on my spare time, too, but I'm a little bit confused...
